ParselyTracker/ParselyTracker.swift

```diff
@@ -210,7 +216,7 @@ public class Parsely {
         track.resume()
     }
     
-    @objc private func suspendExecution() {
+    @objc private func suspendExecution(_notification: Notification) {
```

Comment:
I’d need to brush up and refresh my memory on ObjC `#selector` vs Swift functions exposed to ObjC runtime via `@obj`, but I think that if you update the signature of those two functions there so they take an extra parameter, then the `#selector`s used in your `NotificationCenter.default.addObserver` calls will need to updated to `#selector(suspendExecution:)` and `#selector(resumeExecution:)` (note the extra colon character) to reflect that.

Reply:
Ah indeed! Forgot what the right syntax was, thanks for the correction; it's been ages since I've used ObjC, let alone #selectors 😅

Reply:
> If I'm not using it would it be more correct to remove it

I think opinions might vary, but fwiw, that makes sense to me -- no need to pass it if it's not being used. :)

Reply:
From Apple's official [documentation on `NotificationCenter`'s `addObserver(_:selector:name:object:)` method](https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver):

> The method that aSelector specifies must have one and only one argument (an instance of NSNotification).

Notice the "must". I don't know how strongly they enforce that requirement, and while I know that I've seen ObjC code from years ago use methods and selectors without any parameter for NC observers, it might not be good practice (i.e. maybe it ended up working only by chance, depending on how the parameters get stacked during method call to the inner `performSelector:` / `objc_msgSend` call that likely happens under the hood to notify the observers) and might even have only worked in older (32-bits?) OS versions, while it's now more hardly enforced as a strong requirement or at least more of an error not to accept a parameter for those?

So IMHO, if in doubt, I'd just provide the `_ notification: Notification` parameter to the method signatures—and update the corresponding `#selector` accordingly—even if that `notification` parameter is unused in practice.

Having unused parameters like that was actually a pretty common case for methods implementing an (informal or explicit) protocol in ObjC back in the day (especially thinking about many delegate methods for which we don't use the parameters most of the time, for example), so wouldn't shock me much to see a similar case for `notification` here 🤷  
(And I'm not sure there's an equivalent of C's / ObjC's `__unused` or `__attribute__((unused))` annotation in Swift anyway?)

Reply:
> Notice the "must".

Well spotted! I know cases where we've omitted the argument in the past without issues, but the docs call for it then that settles the matter. :)